templates/Makefile

```diff
@@ -5,16 +5,18 @@ bump-versions: bump-go-versions bump-rust-versions
 bump-go-versions:
 	@for dir in $$(ls -d *-go) ; do \
 		cd $$dir/content ; \
-		sed -i '' -e 's%require github.com/fermyon/spin/sdk/go v.*%require github.com/fermyon/spin/sdk/go ${SDK_VERSION}%g' go.mod ; \
-		sed -i '' -e 's/{{project-name | snake_case}}/foo/g' go.mod ; \
+		sed -i.sed-bak -e 's%require github.com/fermyon/spin/sdk/go v.*%require github.com/fermyon/spin/sdk/go ${SDK_VERSION}%g' go.mod ; \
```

Comment:
Supports a wider array of sed flavors, I presume?

Reply:
That's the idea: https://github.com/fermyon/spin/pull/635/commits/d609eb19d8dd568c471bf1fc947859936e7402cb